pkg/kubelet/kubelet_pods.go

```diff
@@ -78,6 +78,12 @@ const (
 	podKillingChannelCapacity = 50
 )
 
+// Container state reason list
+const (
+	PodInitializing   = "PodInitializing"
```

Comment:
Also use it here: https://github.com/kubernetes/kubernetes/blob/ac101cbdda382d4c23485a3aa5583d79e5379e31/pkg/kubelet/kubelet_test.go#L1458?

Reply:
@SergeyKanzhelev yes, I've modified, thanks!